lib/Basics/files-slurp-decrypt.cpp

```diff
@@ -0,0 +1,105 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2014-2022 ArangoDB GmbH, Cologne, Germany
+/// Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Dr. Frank Celler
+////////////////////////////////////////////////////////////////////////////////
+
+#include "files-slurp-decrypt.h"
+
+#include <fcntl.h>
+#include <sys/stat.h>
+
+#include "Basics/files.h"
+#include "Basics/ScopeGuard.h"
+#include "Basics/StringBuffer.h"
+#include "Enterprise/Encryption/EncryptionFeature.h"
+
+using namespace arangodb;
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief slurps in a file that is encrypted and return unencrypted contents
+////////////////////////////////////////////////////////////////////////////////
+
+/// @brief read buffer size (used for bulk file reading)
+#define READBUFFER_SIZE 8192
```

Comment:
As this define is only used in the following function, can we turn it into a constexpr?

Reply:
Done!